consensus/src/user/provisioners.rs

```diff
@@ -145,91 +137,103 @@ impl Provisioners {
         pubkey_bls: &PublicKey,
         amount: u64,
     ) -> Option<u64> {
-        let stake = self.members.get_mut(pubkey_bls)?;
+        let stake = self.0.get_mut(pubkey_bls)?;
         if stake.value() < amount {
             None
         } else {
             stake.subtract(amount);
             let left = stake.value();
             if left == 0 {
-                self.members.remove(pubkey_bls);
+                self.0.remove(pubkey_bls);
             }
             Some(left)
         }
     }
 
     pub fn remove_stake(&mut self, pubkey_bls: &PublicKey) -> Option<Stake> {
-        self.members.remove(pubkey_bls)
+        self.0.remove(pubkey_bls)
     }
 
     /// Adds a new member with reward=0 and elibile_since=0.
```

Comment:
Here the function name changed from "member" to "provisioner", yet the comment stayed with "member", is it on purpose. Now the comment deviates more from code than before.

Reply:
It's a leftover. Will update it. Thanks